src/fswAlgorithms/orbitControl/lambertSolver/lambertSolver.cpp

```diff
@@ -204,6 +204,44 @@ std::vector<double> LambertSolver::dTdx(double x, double T, double lam)
     return DTs;
 }
 
+/*! This method includes a 3rd order householder root-finder to find the free variable x that satisfies the time-of-flight constraint
+    @param T requested non-dimensional time-of-flight
+    @param x0 initial guess for x free variable of Lambert's problem that satisfies the given time of flight
+    @param N number of revolutions
+    @return std::vector<double>
+*/
+std::vector<double> LambertSolver::householder(double T, double x0, int N)
+{
+    double tol = 1e-8;
+    int iter_max = 8;
+    double xnew = 0.0; // initialize
+    int j;
+    double err;
+
+    for (j = 0; j <= iter_max; ++j){
+        // compute non-dimensional time-of-flight T for given x
+        double tof = x2tof(x0, N, this->lambda);
+        // get derivatives of T
+        std::vector<double> DTs = dTdx(x0, tof, this->lambda);
+        double DT = DTs[0];
+        double D2T = DTs[1];
+        double D3T = DTs[2];
+
+        double delta = tof - T;
+        // compute new x using 3rd order householder algorithm
+        xnew = x0 - delta*(pow(DT,2) - delta*D2T/2.0)/(DT*(pow(DT,2) - delta*D2T) + D3T*pow(delta,2)/6.0);
+
+        err = abs(x0 - xnew);
+        x0 = xnew;
+        if (err < tol){
+            break;
+        }
+    }
+    std::vector<double> sol = {xnew, static_cast<double>(j), err};
```

Comment:
:large_orange_diamond: Depending on what the solution is meant to be, you could also directly return a std::array<double, 3> here.

Reply:
Done.